### DISCUSSION TITLE
VLC won't play dvd

### DISCUSSION THREAD

**Discussion by dacomboman** [SpiralLinux/SpiralLinux-project#372] [General]
Even if all necessary codecs seem to be installed by default, VLC still can't play a DVD.

- - - - -

Btw, have never had issues burning SpiralLinux via Rufus to usb.

**Comment by geckolinux** [comment]
Hi there, is it a commercial encrypted DVD? What DVD device path are you trying? What errors does it throw?

**Comment by marwenius** [comment]
Try to install `libdvd-pkg`:

https://wiki.debian.org/CDDVD#Watching_Video_DVDs
https://www.videolan.org/developers/libdvdcss.html

**Comment by dacomboman** [comment]
What is weird, is the 1st DVD did not even load.
Put in another, and VLC was able to play it.
Put in a different one again, and VLC could not load it.
Three different "commercial" type DVDs.

Codecs installed:
![Screenshot from 2023-12-06 10-51-03](https://github.com/SpiralLinux/SpiralLinux-project/assets/10890984/3fc1b027-39e6-4d5d-b421-57c29c62bf98)

**Reply by geckolinux** [reply]
> Three different "commercial" type DVDs.

Ok, so it sounds like different type of DVD copy protection (encryption), and @marwenius's suggestion https://github.com/orgs/SpiralLinux/discussions/372#discussioncomment-7775745 should work.

**Comment by dacomboman** [comment]
Sorry if i'm not as savvy as you folks are...
I easily get lost in those Debian pages.

Did however find this tutorial and it did the trick:

Install libdvd-pkg Using apt-get

Update apt database with apt-get using the following command.

sudo apt-get update

After updating apt database, We can install libdvd-pkg using apt-get by running the following command:

sudo apt-get -y install libdvd-pkg
 
--How To Install libdvd-pkg on Debian 12 | Installati.one 
--https://installati.one/install-libdvd-pkg-debian-12/

**Reply by dacomboman** [reply]
Btw, did not want to forget to say "thank you" @marwenius for pointing out the need to install that libdvd-pkg and
@geckolinux for insisting that i follow up on that lead.
